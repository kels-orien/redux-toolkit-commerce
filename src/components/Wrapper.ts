import styled from 'styled-components'

const Wrapper = styled.div`
  max-width: 1240px;  
  display: block;
  margin: 0 auto;
`

export default Wrapper

