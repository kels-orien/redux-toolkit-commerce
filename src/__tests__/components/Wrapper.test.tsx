import React from 'react'
import renderer from 'react-test-renderer'
import Wrapper from '../../components/Wrapper'

test('Wrapper render correctly', () => {
    const tree = renderer.create(<Wrapper/>).toJSON();

    expect(tree).toMatchSnapshot()

})