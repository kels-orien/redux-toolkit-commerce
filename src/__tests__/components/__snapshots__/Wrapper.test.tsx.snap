// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper render correctly 1`] = `
<div
  className="sc-bdnxRM lmFpMk"
/>
`;
